R/strategise.R

```diff
@@ -0,0 +1,120 @@
+strategise <- function(options, db) {
+  cost <- function(baseline_settings, intervention_settings, intervention) {
+    population <- baseline_settings$population
+    procurement_buffer <- (intervention_settings$procureBuffer + 100) / 100
+    cost_per_N1 <- intervention_settings$priceNetStandard
+    cost_per_N2 <- intervention_settings$priceNetPBO
+    price_NET_delivery <- intervention_settings$priceDelivery
+    price_IRS_delivery <- intervention_settings$priceIRSPerPerson * population
```

Comment:
Could this be renamed? Got a bit confused thinking it was actually something to do with spray delivery, in the way that we have net delivery costs. I'd be inclined to make this value the price of IRS per person and multiply by population below as you have done with the other cost calculations.

Reply:
Absolutely - can definitely be cleaned up. It's only like this because it's copied verbatim from original code [here](https://github.com/mrc-ide/mintr/blob/master/tests/testthat/test-config.R#L17). Also reason for the redundancy highlighted by your other comment. I'm not sure whether the logic should exist in both places, but I'll tidy it up here, and the endpoint tests are impliclitly checking the calculations.

Reply:
This has been done, and it's definitely an improvement, so thanks for the suggestion. I have added tests for the new cost function that compare its outputs with the authoritative version that was originally provided by Ellie.